phpunit.xml

```diff
@@ -17,4 +17,9 @@
             <directory suffix="Test.php">./tests</directory>
         </testsuite>
     </testsuites>
+    <filter>
```

Comment:
👍🏻 Are you generating code coverage? Or do you just do this as a matter of course on all projects? Good request, either way.

Reply:
Yes. I want to generate the code coverage report to help me improving tests.